src/aat/resources/features/F-047/S-573.td.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "must return negative response HTTP-400 when request contains a non-existing jurisdiction ID",
+  "_guid_": "S-573",
+  "_extends_": "F-047_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "contains a non-existing jurisdiction ID",
+    "code is HTTP-400"
+  ],
+  "request": {
+    "pathVariables": {
+      "jid": "non-existing jurisdiction ID"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "body": {
+      "exception": "[[ANY]]",
+      "message": "Missing start trigger token",
```

Comment:
Seems this message is not a suitable one for this non-existing jurisdiction ID scenario.
Please update the message if you know.

Reply:
done